src/packages/@app/pages/Collections/CollectionPage.ViewModel.ts

```diff
@@ -990,10 +990,18 @@ export default class CollectionsViewModel {
   public handleValidateCurl = async (importCurl: string) => {
     const response =
       await this.collectionService.importCollectionFromCurl(importCurl);
+    const method = await response?.data?.data?.request?.method;
     if (response.isSuccessful) {
-      return true;
+      const isSuccessful = response.isSuccessful;
+      return {
+        isSuccessful: isSuccessful,
```

Comment:
1. shift `const method` inside if block, as we only get `method` when response will success.